src/core/lv_obj_pos.c

```diff
@@ -319,13 +319,26 @@ void lv_obj_align(lv_obj_t * obj, lv_align_t align, int32_t x_ofs, int32_t y_ofs
     lv_obj_set_pos(obj, x_ofs, y_ofs);
 }
 
+static void add_anchored_children(lv_obj_t * obj, const lv_obj_t * base)
+{
+    base->spec_attr->anchored_child_count++;
```

Comment:
Somewhere before we arrive here we need to do one of

- `if(base->spec_attr == NULL) return;`
- `lv_obj_allocate_spec_attr(base);`

May not be in this function, but somewhere, such that the test cases in the CI pass and there's no chance of it being NULL.